mautic/mautic
# A user can edit an email or landing page created in Code Mode in the Builder, after she acknowledged the risk

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:


As a user I might want to change something in the code mode, but then keep editing my email or landingpage in the builder. 

This PR offer not to block that by default, but enable the user to accept the risk that the builder could change the HTML, and let continue in the builder.
#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a new email or use the existing one
3. Choose the Code Mode
4. Click the builder button
5. The warning message here

  ![image](https://user-images.githubusercontent.com/96085911/179037062-a683c297-d72c-4de9-8278-a8bff6335a73.png)
7. Click "ok". The builder is opened
8. Click "cansel". The builder is not opened





<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11339"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
